src/test/java/org/broadinstitute/hellbender/tools/GenomicsDBImportIntegrationTest.java

```diff
@@ -0,0 +1,107 @@
+package org.broadinstitute.hellbender.tools;
+
+import com.intel.genomicsdb.GenomicsDBFeatureReader;
+import htsjdk.tribble.AbstractFeatureReader;
+import htsjdk.tribble.CloseableTribbleIterator;
+import htsjdk.tribble.readers.LineIterator;
+import htsjdk.tribble.readers.PositionalBufferedStream;
+import htsjdk.variant.bcf2.BCF2Codec;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFCodec;
+import org.apache.commons.io.FileUtils;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.tools.genomicsdb.GenomicsDBImport;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.broadinstitute.hellbender.utils.test.GenomicsDBTestUtils;
+import org.broadinstitute.hellbender.utils.test.VariantContextTestUtils;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public final class GenomicsDBImportIntegrationTest extends CommandLineProgramTest {
+
+  private static final String GENOMICSDB_WORKSPACE = createTempDir("genomicsdb-tests-").getAbsolutePath() + "/workspace";
+  private static final String GENOMICSDB_ARRAYNAME = "gatk-genomicsdb-test-array";
+  private static final String TEST_CALLSETMAP_JSON_FILE = GENOMICSDB_WORKSPACE + "/callset.json";
+  private static final String TEST_VIDMAP_JSON_FILE = GENOMICSDB_WORKSPACE + "/vidmap.json";
+
+  private static final String hg00096 = largeFileTestDir + "gvcfs/HG00096.g.vcf.gz";
+  private static final String hg00268 = largeFileTestDir + "gvcfs/HG00268.g.vcf.gz";
+  private static final String na19625 = largeFileTestDir + "gvcfs/NA19625.g.vcf.gz";
+  private static final String combined = largeFileTestDir + "gvcfs/combined.gatk3.7.g.vcf.gz";
+
+  private static final String hg00096_cloud = getGCPTestInputPath() + "large/gvcfs/HG00096.g.vcf.gz";
+  private static final String hg00268_cloud = getGCPTestInputPath() + "large/gvcfs/HG00268.g.vcf.gz";
+  private static final String na19625_cloud = getGCPTestInputPath() + "large/gvcfs/NA19625.g.vcf.gz";
```

Comment:
Move the declaration of these constants (hg00096_cloud, hg00096, etc.) inside your GenomicsDBImporterTestData 
 `DataProvider` below -- it's actually problematic to call `getGCPTestInputPath()` during static class initialization.